tool/tctl/common/bots_command.go

```diff
@@ -140,7 +151,7 @@ func (c *BotsCommand) Initialize(app *kingpin.Application, _ *tctlcfg.GlobalCLIF
 
 // TryRun attempts to run subcommands.
 func (c *BotsCommand) TryRun(ctx context.Context, cmd string, clientFunc commonclient.InitFunc) (match bool, err error) {
-	var commandFunc func(ctx context.Context, client *authclient.Client) error
+	var commandFunc func(ctx context.Context, client authclient.ClientI) error
```

Comment:
Bit of a finicky ask - but generally, I've been trying to slowly kill off these very large centrally defined interfaces since they've caused a lot of pain in the past. Can we just define an unexported interface here with just the methods that we require?